stories/Templates/PublicationPage/PublicationPage.jsx

```diff
@@ -41,94 +41,128 @@ export const PublicationPage = ({
-          <div className="cell small-12 title-mobile">
-            <div className="column">
-              <Heading
-                type="2"
-                label={blockHeader}
-              />
-            </div>
-          </div>
-          <div className="cell small-12 large-5 medium-6 large-offset-1 medium-offset-0">
-            <div className="column sticky">
-              <PublicationCard
-                image={image && <Publicationthumb />}
-                download={download}
-                readmore={readmore}
-              />
-            </div>
-          </div>
-          <div className="cell small-12 large-5 medium-5">
-            <div className="column">
-              <Heading className="title-desktop"
-                type="2"
-                label={blockHeader}
-              />
-              <Heading className="description-text"
-                type="6"
-                label={blockDescription}
-              />
-              <P label={paragraphTop1} className="margin-top-48" />
-              <P label={paragraphTop2} />
-              <P label={paragraphTop3} />
-            </div>
-          </div>
-        </div>
-        <div className="grid-x grid-margin-x margin-top-48 related-publication">
-          <div className="cell small-12 large-6 large-offset-6 border-top-2">
-            <div className="column">
-              <Heading
-                type="2"
-                label={relatedPublicationHeading}
-              />
-            </div>
-          </div>
-          <div className="cell small-12 large-6 large-offset-6">
-            <div className="column">
-              <ContentCardWithOutImage className="card-out"
-                Emphasize="false"
-                data={ContentCardWithOutImageData}
-              />
+            <div class="grid-x grid-margin-x publication-block">
+                <div class="cell small-12 title-mobile">
+                    <div class="column">
+                        <Heading
+                            type='2' 
+                            label={blockHeader} 
+                        />
+                    </div>
+                </div>
+                <div class="cell small-12 large-5 medium-6 large-offset-1 medium-offset-0">
+                    <div class="column sticky">
+                        <PublicationCard
+                            image={image && <Publicationthumb/>}
+                            download={download}       
+                            readmore={readmore}                 
+                        />
+                    </div>
+                </div>
+                <div class="cell small-12 large-5 medium-5">
+                    <div class="column">
+                        <Heading className="title-desktop"
+                            type='2' 
+                            label={blockHeader} 
+                        />
+                        <Heading className="description-text"
```

Comment:
@neelesh-tippe Change the classname `description-text` to `publication-card__date`. Also any specific reason you are using header tag for date or can you change this?
Also can you remove this class and use the parent class to style both the title and this date element?

Reply:
changes done

Reply:
Header tag used because it is mentioned in figma